packages/frontend/app/root.tsx

```diff
@@ -101,6 +128,7 @@ export default function App() {
 export function ErrorBoundary() {
   const error = useRouteError()
 
+  // TODO: is there a better approach for handling the logout Url on the ErrorPage given that the user may / may not eb logged in, and we don'tr have access to a loader.
```

Comment:
I realize this doesn't solve the problem but I don't think we should show the sidebar at all when there is an error and they're not authenticated. It's the only way they can see the sidebar without authenticating. Could probably argue the information leaking doesn't matter (tab names) but it's also just inconsistent/unexpected from a UI perspective IMO.  

I think what you have is reasonable but I also think it would be reasonable to never show the sidebar (even when authenticated) in service of hiding it when unauthenticated. 

It does make you wonder how we can get logged in state client side. Since they're http only cookies we cant check for them, so I guess we would need to track it and save on the context or something. Or perhaps we could make a request here in the error page, but I don't think that's worth it.

Reply:
I don't think the error boundary supports async calls. The better approach seems to be removing the sidebar since all error pages have a `Go to homepage` button. So this will resolve appropriately whether the user is logged in or not and there is a cl;ear option to click for someone stuck on an error page without the sidebar to use for navigation. Is that okay?

Reply:
seems good to me